lib/converters/converter-v2-to-v1.js

```diff
@@ -59,6 +59,28 @@ _.extend(Builders.prototype, {
     },
 
     /**
+     * Extracts all requests from a v2 collection or folder
+     *
+     * @param folderOrCollection
+     */
+    extractItems: function (folderOrCollection) {
```

Comment:
@vjrngn This is in context of the V2 [`Items`](http://www.postmanlabs.com/postman-collection/Item.html), not any generic items.